### DISCUSSION TITLE
reServer Industrial | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1164] [Q&A]
# reServer Industrial | Seeed Studio Wiki

reServer Industrial Getting Started

https://wiki.seeedstudio.com/reServer_Industrial_Getting_Started/

**Comment by paymentinapp** [comment]
I purchased a reServer industrial J3010( order#4000175171). 
1. I would like to know a CCTV camera connected on device LAN port. How can I know the IP address of the camera?
2. I understood CCTV can get power via POE. Do I need to configure it? When I try it based on below link, There is no 'gpio' folder(cd /sys/class/gpio)
https://wiki.seeedstudio.com/reServer_Industrial_Getting_Started/#gigabit-ethernet-connectors

**Reply by qunanan** [reply]
You have to manually set the IP address to access the camera, I think you need to check the camera manual for more details.

**Comment by park-jihoon-aiv** [comment]
I have purchased a Reserver product and would like to power it using the PoE LAN port. However, when I run the command `sudo cat /sys/kernel/debug/gpio` as mentioned in the manual, I cannot see gpiochip2 (300-315), making it impossible to configure. Could you advise me on how to resolve this issue?

**Reply by ZhuYaoHui1998** [reply]
which Jetpack in your reserver device?  Jetpack 5.1.1 or Jetpack 6.0. and which device did you purchase?  j3010 J3011 J4011 J4012 or J2302

**Reply by park-jihoon-aiv** [reply]
The device is J4012 and Jetpack is 5.1.3-b29
R35 (release), REVISION: 5.0

**Reply by jjjadand** [reply]
hi, park. I tested it based on reServer j4012 && jetpack 5.1.3 and it is normal. Can you take a screenshot and give me more details?
![ad367d7ba1a7debd8721dd209d9aea5f_compress](https://github.com/user-attachments/assets/0608d1dc-897e-4862-89b7-f80dc46f0b56)
